sacro/adapters/local_audit.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import os
+from pathlib import Path
+
+
+def get_filename():
+    filename = Path(os.getenv("APPDATA", ".")) / "SACRO" / "audit.log"
+    filename.parent.mkdir(parents=True, exist_ok=True)
+    return str(filename)
+
+
+def make_logger(filename):
+    handler = logging.FileHandler(filename=filename)
+    formatter = logging.Formatter(
+        "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
```

Comment:
nit: not very keen on this format, as it is not very machine readable due the spaces.

Reply:
Does it need to be machine readable? I don't think we have that requirement at the moment.

Do you have a suggestion that's still easily human readable?

Reply:
We don't have a strict requirement.

We have an idea that we show the log in the browser - might be nice to do more than plain text for that.

This is a nit, not a blocker from my perspective.

But I think key=value works well for filtering logs, and is self documenting

e.g. sthlike
```
2023-06-27 08:23:19,215 - audit - INFO - "File released" user=user file=crosstab_fail_2_2023-06-20-08504244```

Also, I just noted the European time format, might be worth ISO?

Again, nits, feel free to ignore.